### ISSUE TITLE
AstropyDeprecationWarning: The rotateCD function is deprecated

### ISSUE THREAD

**Issue by pllim** [spacetelescope/drizzlepac#21]
Example test: https://ssb.stsci.edu/pandokia/pandokia.cgi?query=detail&key_id=36464377

```
[WARNING ] /srv/iraf/bldtmp/porcelain.g23AA24Tm8/porcelain/envs/dev/lib/python2.7/site-packages/drizzlepac/wcs_functions.py:592:
AstropyDeprecationWarning: The rotateCD function is deprecated and
may be removed in a future version.
  outwcs.rotateCD(_delta_rot)
```

**Comment by mcara** [comment]
Not only ``rotateCD`` has been deprecated but also it has been modified from its original definition (upon which our code relied) in https://github.com/astropy/astropy/pull/5189.

PR https://github.com/spacetelescope/drizzlepac/pull/26 addresses these issues.

**Comment by mcara** [comment]
Solved in #26
